app/src/main/java/com/zulip/android/networking/response/UserConfigurationResponse.java

```diff
@@ -0,0 +1,246 @@
+package com.zulip.android.networking.response;
+
+import com.google.gson.annotations.SerializedName;
+import com.zulip.android.models.Person;
+import com.zulip.android.models.Stream;
+import com.zulip.android.models.updated.ChatStatusWrapper;
+import com.zulip.android.models.updated.JenkinsType;
+import com.zulip.android.models.updated.Referrals;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by patrykpoborca on 8/25/16.
+ */
+
+public class UserConfigurationResponse {
+
+    @SerializedName("last_event_id")
+    private int lastEventId;
+
+    @SerializedName("realm_domain")
+    private String realmDomain;
+
+    @SerializedName("realm_name")
+    private String realmName;
+
+    @SerializedName("result")
+    private String result;
+
+    @SerializedName("realm_default_language")
+    private String realmDefaultLanguage;
+
+    @SerializedName("presences")
+    private Map<String, ChatStatusWrapper> presences;
+
+    @SerializedName("pointer")
+    private int pointer;
+
+    @SerializedName("realm_create_stream_by_admins_only")
+    private boolean realmCreateStreamByAdminsOnly;
+
+    @SerializedName("email_dict")
+    private Map<String, String> emailDict;
+
+    @SerializedName("msg")
+    private String msg;
+
+    @SerializedName("realm_allow_message_editing")
+    private boolean realmAllowMessageEditing;
+
+    @SerializedName("realm_emoji")
+    private Map<String, JenkinsType> realmEmoji;
+
+    @SerializedName("default_language")
+    private String defaultLanguage;
+
+    @SerializedName("realm_restricted_to_domain")
+    private boolean realmRestrictedToDomain;
+
+    @SerializedName("max_message_id")
+    private int maxMessageId;
+
+    @SerializedName("queue_id")
+    private String queueId;
+
+    @SerializedName("realm_message_content_edit_limit_seconds")
+    private int realmMessageContentEditLimitSeconds;
+
+    @SerializedName("realm_invite_by_admins_only")
+    private boolean realmInviteByAdminsOnly;
+
+    @SerializedName("realm_invite_required")
+    private boolean realmInviteRequired;
+
+    @SerializedName("referrals")
+    private Referrals referrals;
+
+    @SerializedName("twenty_four_hour_time")
+    private boolean twentyFourHourTime;
+
+    @SerializedName("left_side_userlist")
+    private boolean leftSideUserlist;
+
+    //todo
+    @SerializedName("muted_topics")
+    private List<List<String>> mutedTopics;
+
+    //todo
```

Comment:
these lists are `List<?>` meaning I couldn't figure out their signatures for proper modeling. I got the muted topics signature later, so i can remove that, but not the other's just yet.